.github/workflows/stale.yaml

```diff
@@ -0,0 +1,24 @@
+name: Close Stale Pull Requests
```

Comment:
@iprasannamb , this change is from my previous PR. it shouldn't be here. ensure you are properly updating your branch pls